views/postgres/create.erb

```diff
@@ -189,6 +200,28 @@
                 </div>
               </div>
 
+              <% if @flavor == PostgresResource::Flavor::PARADEDB %>
+                <div class="col-span-full">
+                  <div class="space-y-2">
+                    <label class="text-sm font-medium leading-6 text-gray-900">Partnership Notice</label>
+                    <p class="mt-1 text-sm leading-6 text-gray-600">
+                      This database managed by ParadeDB team. It's powered up with ParadeDB extension.<br>
+                      You can join ParadeDB's community on <a href="https://paradedb.com/community" target="_blank" class="text-orange-600 font-semibold">Slack</a> or ask for help at <a href="mailto:support@paradedb.com" class="text-orange-600 font-semibold">support@paradedb.com</a>.<br>
+                      We will share your details with ParadeDB team. You have to accept ParadeDB's <a href="https://paradedb.com/community" target="_blank" class="text-orange-600 font-semibold">Terms of Service</a> and <a href="https://paradedb.com/community" target="_blank" class="text-orange-600 font-semibold">Privacy Policy</a> to be able to create this database.
+                    </p>
+                    <%== render(
+                      "components/form/checkbox",
+                      locals: {
+                        name: "accept_partnership",
+                        options: [
+                          ["1", "Accept Terms of Service", {}, {required: true}],
```

Comment:
It would be good to link to related policies here. Also I think we can combine them into one checkbox.

Reply:
<img width="1512" alt="Screenshot 2024-09-24 at 21 01 22" src="https://github.com/user-attachments/assets/d39c089e-631a-4297-b91c-bc9ccd5788b4">